scripts/local-registry/config.yml

```diff
@@ -0,0 +1,31 @@
+# path to a directory with all packages
+storage: ../../tmp/local-registry/storage
+
+listen: localhost:4872
```

Comment:
`listen: 127.0.01:4872` to avoid Node resolving to IPv6 address

Reply:
changed.